python/semantic_kernel/core_plugins/file_io_plugin.py

```diff
@@ -1,15 +1,17 @@
 # Copyright (c) Microsoft. All rights reserved.
 
 import os
-import typing as t
+import sys
 
 import aiofiles
 
-from semantic_kernel.kernel_pydantic import KernelBaseModel
-from semantic_kernel.plugin_definition import kernel_function, kernel_function_context_parameter
+if sys.version_info > (3, 8):
```

Comment:
Do we know if this is fixed in the latest pydantic version? Or is this not related to Pydantic and we always have to include these Annotated checks?

Reply:
no this is not related to pydantic, Python 3.9 has made Annotated part of typing, typing_extentions is a way to backport a feature like this, which means until oct (eol py3.8) we will have this, luckily a user does not have to think about this, they use the one for their python version.